cabal.haskell-ci

```diff
@@ -2,17 +2,8 @@ no-tests-no-benchmarks: False
 unconstrained:          False
 irc-channels:           irc.freenode.org#haskell-lens
 irc-if-in-origin-repo:  True
+hlint:                  True
 -- The doctests require relatively fancy type-level tricks
 tests:                  >=7.8
 -- Work around https://github.com/haskell/haddock/issues/679
 haddock:                <8.2 || >=8.4
-
--- Because we use a custom Setup script, --test-show-details can only support
--- the options that come with the version of Cabal that is linked against.
--- Unfortunately, --test-show-details=direct wasn't introduced until
--- Cabal-1.24.*, and GHC 7.10 and earlier ship with an older version of Cabal
--- than that. As a result, trying to pass --test-show-details=direct won't work
--- on these GHC versions unless we rebuild Cabal from scratch, which is
--- time-consuming. As a result, we simply disable --test-show-details=direct
--- as a quick-but-dirty workaround.
-test-output-direct:     False
```

Comment:
Hm... I don't think this is actually running `cabal-docspec`?

Reply:
There weren't any. Now there one. Probably good to have something running the code.

Reply:
Ah, OK! I was misled since the title of the PR was just "Cabal docspec", which led to me to gloss over the fact that this PR was originally getting rid of that code path :)

In any case, I like the example you added. Let's land this.